packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -358,6 +358,15 @@ const getStyles = (theme: GrafanaTheme2) => {
       display: 'flex',
       flexDirection: 'column',
 
+      '> *': {
+        zIndex: 0,
+      },
```

Comment:
this prevents tooltips from going under things like RadioButtonGroup such as the ones in those in the Explore graph header, where the options have explicit z-index of 1 or 3. we also have explicit z-index on form Input elements which makes them render over the tooltip when it overlaps an adjacent panel.